test/tests/ghost-basics/run.sh

```diff
@@ -27,4 +27,10 @@ _request() {
 . "$dir/../../retry.sh" '_request GET / --output /dev/null'
 
 # Check that /ghost/ redirects to setup (the image is unconfigured by default)
-_request GET '/ghost/' -I | grep -q '^Location: .*setup'
+if [[ $serverImage == ghost:1* ]]
```

Comment:
Maybe instead pull the full ghost version from the image since the test could run on `ghost:latest`
```bash
ghostVersion="$(docker inspect --format '{{range .Config.Env}}{{ . }}{{"\n"}}{{end}}' "$serverImage" | awk -F= '$1 == "GHOST_VERSION" { print $2 }')"
case "$ghostVersion" in
	0.*) _request GET '/ghost/' -I | grep -q '^Location: .*setup' ;;
	*)   _request GET '/ghost/api/v0.1/authentication/setup/' | grep -q 'status":false' ;;
esac
```